### ISSUE TITLE
[Feature Request] Import SVG from file.

### ISSUE THREAD

**Issue by se4u** [evolus/pencil#452]
* Operating system: Mac OSX 10.11.6
* Pencil version: 3.0.4

It will be great if either of the three options is implemented. 
a.) Either an option to import SVG from file system to a collection.
b.) A helper script to convert an SVG or a folder of SVGs into a collection.
c.) The search results on `openclipart.org` return the first page instead of the last page because that makes the search useless.

Since the ability to import cliparts from openclipart.org is already built into pencil therefore option b should be easy to do and it does not require recompiling pencil since an external script is created. I am willing to work on this feature but I couldn't really figure out where the javascript code to convert an svg into a collection exists?

**Comment by se4u** [comment]
I just found the SVG stencil builder which seems like it should do the job but I can't figure out how to copy an external SVG into the stencil.

**Comment by defbrett** [comment]
Bump for this. 
Drag and drop import of .svg files and edit <svg> properties just like version 2.x 
I have entire collections of artifacts that have .svg's in them and I have entire libraries of .svg's that I'm not particularly fond of having to convert them all to .png or bitmap to use 3.x. Finally, the quality of an .svg and ease of tweaking colors, opacity and scaling is easier or at least was easier in 2.x 
Am I supposed to now have a .png/.bmp icon of every color, size, and shade combination I could ever need? No thanks I'll stay on 2.x
But i guess based on the lack of responses to this feature I'm the only dummy editing <svg> code for tweaks like this..

**Comment by crash83k** [comment]
Yeah. I'm just getting started with Pencil after Gliffy started giving me grief. But the lack of ability to add SVGs to a collection is pretty disappointing. I wish I could help to make it work, but I don't know the language necessary.

**Comment by Cylinder** [comment]
I figured out how to get my SVGs into Pencil.  It is very kludgey.  (I am running 3.0.4 currently).    Click on "Clipart", search for something random on OpenClipart.org, drag a random svg onto the canvas.   Right click on that random svg, click on "Action", then go to "Edit with external editor...".  This will open the svg in Inkscape (by default).    In Inkscape delete the graphic, then go to File, then Import, and select the SVG that you want, and import it.     Go to File-> Document Properties, and resize the page to the size of the drawing.  Then hit the save button and exit.   Tada! your svg is in Pencil.

**Comment by benwinding** [comment]
Another easier way (as the OpenClipart search isn't always available):

1. Manage collections > Install from repository > Install "Stencil Builder"
2. Collections > Add SVG
![Selection_013](https://user-images.githubusercontent.com/11782590/61026123-c50e3800-a3f1-11e9-941f-876ee1f9a0de.png)
3. Select image from file:
![Selection_015](https://user-images.githubusercontent.com/11782590/61026295-128aa500-a3f2-11e9-81be-78826fbb8d7e.png)
3. Change color mode to As-is:
![Selection_014](https://user-images.githubusercontent.com/11782590/61026307-16b6c280-a3f2-11e9-828f-eeb4d2420a4a.png)

Have your SVG and eat it too!

**Comment by adlorenzo30** [comment]
> Another easier way (as the OpenClipart search isn't always available):
> 
>     1. Manage collections > Install from repository > Install "Stencil Builder"
> 
>     2. Collections > Add SVG
>        ![Selection_013](https://user-images.githubusercontent.com/11782590/61026123-c50e3800-a3f1-11e9-941f-876ee1f9a0de.png)
> 
>     3. Select image from file:
>        ![Selection_015](https://user-images.githubusercontent.com/11782590/61026295-128aa500-a3f2-11e9-81be-78826fbb8d7e.png)
> 
>     4. Change color mode to As-is:
>        ![Selection_014](https://user-images.githubusercontent.com/11782590/61026307-16b6c280-a3f2-11e9-828f-eeb4d2420a4a.png)
> 
> 
> Have your SVG and eat it too!

Hi, I followed your steps but the SVG is always imported black, don't matter if I change the color in Inkscape

**Comment by benwinding** [comment]
Still works for me on version `3.1.0` What version are you using?

![image](https://user-images.githubusercontent.com/11782590/82163869-c7340380-98ec-11ea-94ad-98d41f6c49ad.png)

**Comment by adlorenzo30** [comment]
> Still works for me on version `3.1.0` What version are you using?
> 
> ![image](https://user-images.githubusercontent.com/11782590/82163869-c7340380-98ec-11ea-94ad-98d41f6c49ad.png)

The same version, maybe is a bug in Mac

**Comment by dgthanhan** [comment]
Have you tried converting the Inkscape object to path first by Path > Object to Path?
I have no problem with it after converting, using the standard Pencil installation without even the Stencil Builder, which is not intentionally created for this purpose.

![image](https://user-images.githubusercontent.com/7098377/82169334-62c47480-98eb-11ea-9657-425a69de3299.png)

**Comment by adlorenzo30** [comment]
Well, look like a problem with that SVG, a tried with others and work. Thanks man for everything
